parity/params.rs

```diff
@@ -64,6 +65,7 @@ impl str::FromStr for SpecType {
 			"musicoin" => SpecType::Musicoin,
 			"ellaism" => SpecType::Ellaism,
 			"easthub" => SpecType::Easthub,
+			"ethereumsocial" => SpecType::Ethereumsocial,
```

Comment:
ethereum-social? EthereumSocial?

Reply:
@5chdn EthereumSocial for capital, ethereumsocial for small letters 😄